packages/cashier/src/types/routes.types.ts

```diff
@@ -4,11 +4,11 @@ export type TRoute = {
     default?: boolean;
     exact?: boolean;
     id?: string;
-    icon_component: string;
+    icon_component?: string;
     is_invisible?: boolean;
     path?: string;
     to?: string;
-    component?: typeof React.Component;
+    component?: React.FC<any>;
```

Comment:
Here `component` can be Deposit.tsx, Withdrawal.tsx, PaymentAgent.tsx, AccountTransfer.tsx, PaymentAgentTransfer.tsx, P2P.tsx, OnRamp.tsx or Cashier.tsx. All of these components have required and optional properties. If we want replace `any` with the type of properties of above components, some of them can start complain that there is no required property in particular component. For example in AccountTransfer there is required onClose method, and we should add this method to other components too to satisfy TS, but this prop really doesn't needed for other components. One of the solution is to make all properties `optional`, but:
1. Because properties become optional we need to check this properties not to be undefined in every component (Deposit, Withdrawal, etc.)
2. IMHO there is no any benefit here to type properties for these components